taskt/Core/Automation/Commands/IE Browser Commands/IENavigateToURLCommand.cs

```diff
@@ -1,78 +1,74 @@
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_InstanceName { get; set; }
+
         [XmlAttribute]
-        [Attributes.PropertyAttributes.PropertyDescription("Please Enter the URL to navigate to")]
-        [Attributes.PropertyAttributes.InputSpecification("Enter the destination URL that you want the IE instance to navigate to")]
-        [Attributes.PropertyAttributes.SampleUsage("https://mycompany.com/orders")]
-        [Attributes.PropertyAttributes.Remarks("")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
+        [PropertyDescription("Navigate to URL")]
+        [InputSpecification("Enter the destination URL that you want the IE instance to navigate to.")]
+        [SampleUsage("https://example.com/ || {vURL}")]
+        [Remarks("")]
+        [PropertyUIHelper(PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
         public string v_URL { get; set; }
 
         public IENavigateToURLCommand()
         {
-            this.CommandName = "IENavigateToURLCommand";
-            this.SelectionName = "Navigate to URL";
-            this.v_InstanceName = "default";
-            this.CommandEnabled = true;
-            this.CustomRendering = true;
+            CommandName = "IENavigateToURLCommand";
+            SelectionName = "Navigate to URL";
+            v_InstanceName = "default";
+            CommandEnabled = true;
+            CustomRendering = true;
         }
 
         public override void RunCommand(object sender)
         {
             object browserObject = null;
 
-            var engine = (Core.Automation.Engine.AutomationEngineInstance)sender;
+            var engine = (AutomationEngineInstance)sender;
 
             var vInstance = v_InstanceName.ConvertToUserVariable(engine);
-
             browserObject = engine.GetAppInstance(vInstance);
-
-            var browserInstance = (SHDocVw.InternetExplorer)browserObject;
+            var browserInstance = (InternetExplorer)browserObject;
 
             browserInstance.Navigate(v_URL.ConvertToUserVariable(sender));
-
             IECreateBrowserCommand.WaitForReadyState(browserInstance);
-
         }
+
         public override List<Control> Render(frmCommandEditor editor)
         {
             base.Render(editor);
 
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_InstanceName", this, editor));
-
             RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_URL", this, editor));
 
             return RenderedControls;
         }
 
         public override string GetDisplayValue()
         {
-            return base.GetDisplayValue() + " [URL: '" + v_URL + "', Instance Name: '" + v_InstanceName + "']";
+            return base.GetDisplayValue() + $" [Navigate to '{v_URL}' Through IE Instance '{v_InstanceName}']";
```

Comment:
please change to [Navigate to '{v_URL}' - Instance Name '{v_InstanceName}']

Reply:
done